<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingwindow</class>
 <widget class="QWidget" name="settingwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>setting</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>571</width>
     <height>350</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_chart">
    <attribute name="title">
     <string>chart</string>
    </attribute>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>555</width>
       <height>47</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>600</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>11</number>
      </property>
      <item>
       <widget class="QLabel" name="label_max_Xaxis">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>maxXaxis</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_max_Xaxis">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_min_Xaxis">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>minXaxis</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_min_Xaxis">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_max_Yaxis">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>maxYaxis</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_max_Yaxis">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_min_Yaxis">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>min_Yaxis</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_min_Yaxis">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>555</width>
       <height>47</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>600</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_pointSize">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>pointSize</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_pointSize">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_fontSize">
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>fontSize</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_fontSize">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_colWidth">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>ColWidth</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_colWidth">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_radarCnt">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>radarCnt</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_radarCnt">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>555</width>
       <height>47</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>600</width>
       <height>16777215</height>
      </size>
     </property>
     <widget class="QLabel" name="label_carWidth">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>11</y>
        <width>70</width>
        <height>19</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>carWidth</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_carWidth">
      <property name="geometry">
       <rect>
        <x>88</x>
        <y>11</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_carLenght">
      <property name="geometry">
       <rect>
        <x>145</x>
        <y>11</y>
        <width>70</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>carLenght</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_carLenght">
      <property name="geometry">
       <rect>
        <x>222</x>
        <y>11</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_mode">
    <attribute name="title">
     <string>mode</string>
    </attribute>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>266</width>
       <height>47</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>600</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QRadioButton" name="radioButton_LCA">
        <property name="text">
         <string>LCA</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_HDS">
        <property name="text">
         <string>HDS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_AUTODOOR">
        <property name="text">
         <string>AUTODOOR</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget_5" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>555</width>
       <height>47</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QComboBox" name="comboBox_chartUp"/>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_chartDown"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_camera">
    <attribute name="title">
     <string>camera</string>
    </attribute>
    <widget class="QComboBox" name="comboBox_camera_1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>150</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_format_1">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>371</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_camera_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>150</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_format_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>60</y>
       <width>371</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_refresh">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>refresh</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_save">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>360</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>save</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
